### PR TITLE
Add the deletion process

### PR DESCRIPTION
**Pull request by lucasfurno**
Include the steps to delete a domain

### CODE REVIEW

**Review by amandapereirasantos — APPROVED** [review]
Everything ok!

File: products/edge-application/domains/2021-01-14-index.md [hunk 1]
```diff
@@ -85,4 +86,14 @@ Edge Application: select the edge application which will receive the domain.
 
 ---
 
+## 6. Deleting {#deleting}
+
+To delete Domains, do as follows:
```
Comment (Luize-boyen) [inline products/edge-application/domains/2021-01-14-index.md:91]: To delete Domains follow the steps below:
or
To delete Domains proceed as follows:

File: products/edge-application/domains/2021-01-14-index.md [hunk 2]
```diff
@@ -85,4 +86,14 @@ Edge Application: select the edge application which will receive the domain.
...
+
+1. Access [Real-Time Manager](https://manager.azion.com/) and select **Domains** in the Edge Computing menu.
+2. Locate the Domain that you need to remove and click on **Trash icon** on the right-side
```
Comment (Luize-boyen) [inline products/edge-application/domains/2021-01-14-index.md:94]: 2. Select the Domain you want to remove and click the **Trash icon** on the right;

File: products/edge-application/domains/2021-01-14-index.md [hunk 3]
```diff
@@ -85,4 +86,14 @@ Edge Application: select the edge application which will receive the domain.
...
+3. Confirm the deletion typing **delete** and click **Delete**
```
Comment (Luize-boyen) [inline products/edge-application/domains/2021-01-14-index.md:95]: Confirm the deletion typing **delete** and click **Delete**. (só o ponto final)

File: products/edge-application/domains/2021-01-14-index.md [hunk 4]
```diff
@@ -85,4 +86,14 @@ Edge Application: select the edge application which will receive the domain.
 
 ---
 
+## 6. Deleting {#deleting}
+
+To delete Domains, do as follows:
+
+1. Access [Real-Time Manager](https://manager.azion.com/) and select **Domains** in the Edge Computing menu.
```
Comment (Luize-boyen) [inline products/edge-application/domains/2021-01-14-index.md:93]: 1. Access [Real-Time Manager](https://manager.azion.com/) and select **Domains** in the Edge Computing menu; (ponto e vírgula)

**Review by Luize-boyen — CHANGES_REQUESTED** [review]

File: products/edge-application/domains/2021-01-14-index.md (hunk 2 above)
Comment (lucasfurno) [reply products/edge-application/domains/2021-01-14-index.md:94]: Does not have an option to "Select". The icon appears when I point the cursor on the right

Comment (Luize-boyen) [reply products/edge-application/domains/2021-01-14-index.md]: In this case I suggest using Locate as you said before.

**Review by Luize-boyen — APPROVED** [review]

**Pull request merged**